generator_process/actor.py

```diff
@@ -38,6 +38,7 @@ def __init__(self):
         self._exception = None
         self.done = False
         self.cancelled = False
+        self.actor = actor
```

Comment:
Where is the actor local variable it is assigning coming from?

Reply:
Forgot to remove it. I was thinking of having the future instantly send the cancel message instead of having it wait on the send loop. Current implementation will run another step before canceling but I don't think that's too big of an issue to worry about.

Reply:
Ah ok. Yes, I'm fine with it being off a step.